common/lib/symbiosis/domain_skeleton.rb

```diff
@@ -0,0 +1,100 @@
+require 'symbiosis/hooks'
+require 'symbiosis/utils'
+require 'pathname'
+
+module Symbiosis
+  # Manages copying a domain skeleton into a freshly-made domain
+  class DomainSkeleton
+    attr_reader :skel_dir
+
+    def initialize(skel_dir = Symbiosis.path_in_etc('symbiosis', 'skel'))
+      @skel_dir = skel_dir
+    end
+
+    def params
+      Dir.glob(File.join(@skel_dir, '**', '*'))
+         .select { |f| File.file?(f) }
+    end
+
+    def should_populate?(domain)
+      Dir.mkdir File.join(domain.directory, 'config')
```

Comment:
This directory will get created by with the owner/group of the user running the process.  Usually this will be root, so an `File.lchown` is required after creation to change the ownership to the domain's uid/gid.